index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Salmon Cookies</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <header class = "top">
+    <h1>Salmon Cookies</h1>
+    			<p id= "title">cookies to go with your coffee</p>
```

Comment:
Be careful about your indentation! Also, don't put spaces on either side of an equals sign in html.